src/styles/block/card.scss

```
@@ -0,0 +1,130 @@
+.main {
+  @include display-justify-content;
+}
+
+.catalog {
+  width: 1024px;
+  @include display-justify-content;
+  flex-wrap: wrap;
+  padding: 0 40px;
+  margin: 0 auto;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  padding: 15px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid $c-border;
+  border-radius: 5px;
+  &--pos {
```

Розділяй селектори відступами для кращої читаємості коду
```suggestion
  border-radius: 5px;
  
  &--pos {
```